packages/sqlite-storage/example/ios/Podfile

```
@@ -0,0 +1,15 @@
+require_relative '../../node_modules/react-native-test-app/test_app'
```

Monorepo friendlier:
```suggestion
ws_dir = Pathname.new(__dir__)
ws_dir = ws_dir.parent until
  File.exist?("#{ws_dir}/node_modules/react-native-test-app/test_app.rb") ||
  ws_dir.expand_path.to_s == '/'
require "#{ws_dir}/node_modules/react-native-test-app/test_app.rb"
```